code/game/machinery/adv_med.dm

```diff
@@ -103,13 +103,18 @@
 		return
 	if(!ismob(O))
 		return
-	var/mob/M = O
+	var/mob/living/M = O//Theres no reason this shouldn't be /mob/living
 	if (src.occupant)
 		user << "\blue <B>The scanner is already occupied!</B>"
 		return
 	if (M.abiotic())
 		user << "\blue <B>Subject cannot have abiotic items on.</B>"
 		return
+
+	if (M.buckled && istype(M.buckled, /obj/structure))//We must make sure the person is unbuckled before they go in
+		var/obj/structure/LB = M.buckled
+		LB.user_unbuckle_mob(user)
```

Comment:
This is actually DNM atm, I should have noted this earlier. Coding principle for SS13 is: the game never does things like this for you. Notice how in all of the lines above, it gives the user feedback and returns, demanding that the user fix the error.

By having the game automatically unbuckle the dude, this principle is broken. Instead, do something like:
```
user << "<span class='warning'>You cannot do that! [M] is buckled to [M.buckled]!</span>"
return
```